changelogs/fragments/79420-galaxy-manifest-license.yaml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - galaxy - include ``license_file`` in the default manifest directives
```

I'd follow `wheel`'s example and use a plural version of the directive name, accepting a list of globs, not just a string.

---

I'm confused about what you're suggesting. I'm not adding a new field. `license_file` accepts a single value and this PR makes sure that file is always included.

---

https://github.com/ansible/ansible/blob/1bda6750f5f4fb8b01de21d1949b02d7547ff838/lib/ansible/galaxy/data/collections_galaxy_meta.yml#L58-L63

---

Ah, I got confused. Maybe this would be better:
```suggestion
  - galaxy - include the file specified using the ``license_file`` in the default manifest directives
```